chart/otomi/values-minimal.yaml

```diff
@@ -0,0 +1,5 @@
+# These values are intended for use to demo Otomi on a server/laptop.
+# Minimal recommended specs: 32GB+ RAM, 12+ (v)CPU
+cluster:
+  provider: onprem
```

Comment:
I can't remove this prop yet, `binzx/bootstrap` will throw an error (see https://github.com/redkubes/unassigned-issues/issues/355)

Reply:
ok